run.ts

```diff
@@ -2,6 +2,8 @@
 
 import { execSync, ExecException } from "child_process";
 import readline from 'readline';
+import fs from 'fs';
+import os from 'os';
```

Comment:
Nit: sort imports in alphabetical order :) See [Google style guide](https://google.github.io/styleguide/jsguide.html#es-module-imports).

Reply:
Thanks for the suggestion. I had a look through the style guide and can't see any recommendation on alphabetically sorting imports. I think if I sort it should be alphabetically on the module name, do you agree? 

Module names are less likely to change whereas the imported method names are more prone to change.